scripts/release.sh

```diff
@@ -76,6 +76,11 @@ git push origin $tag
 # Requires quay credentials in the environment, DPL_DOCKER_REGISTRY_USER, DPK_DOCKER_REGISTRY_KEY
 if [ -z "$debug" ]; then
     make build publish
+    export KFPv2=1
```

Comment:
Why do we care about quay credentials here? Its publishing to PyPi

Reply:
yes, i removed this from the renamed release-branch.sh script which now does no publishing :)